### PR TITLE
feat: adapt package to general PyAnsys repository layout

### PR DESCRIPTION
**Pull request by Revathyvenugopal162**
- Add authors file and adapt to new https://github.com/ansys/pyansys-dev-guide/pull/460
- add contributors
- update precommit config

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+## Individual Contributors
+
+* [Alexander Kaszynski](https://github.com/akaszynski)
+* [Andy Grigg](https://github.com/Andy-Grigg)
+* [Dipin](https://github.com/dipinknair)
+* [Dominik Gresch](https://github.com/greschd)
+* [German](https://github.com/germa89)
+* [Guillem Barroso](https://github.com/GuillemBarroso)
+* [jleonatti](https://github.com/jleonatti)
```
Comment (RobPasMue) [inline CONTRIBUTORS.md:14]:
```suggestion
* [Jess Leonatti](https://github.com/jleonatti)
```

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+## Project Lead
+* [Revathy Venugopal](https://github.com/Revathyvenugopal162)
+
+## Individual Contributors
+
+* [Alexander Kaszynski](https://github.com/akaszynski)
+* [Andy Grigg](https://github.com/Andy-Grigg)
+* [Dipin](https://github.com/dipinknair)
+* [Dominik Gresch](https://github.com/greschd)
+* [German](https://github.com/germa89)
```
Comment (RobPasMue) [inline CONTRIBUTORS.md:12]:
```suggestion
* [German Martínez](https://github.com/germa89)
```

File: CONTRIBUTORS.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Contributors
+
+## Project Lead
+* [Revathy Venugopal](https://github.com/Revathyvenugopal162)
+
+## Individual Contributors
+
+* [Alexander Kaszynski](https://github.com/akaszynski)
+* [Andy Grigg](https://github.com/Andy-Grigg)
+* [Dipin](https://github.com/dipinknair)
```
Comment (RobPasMue) [inline CONTRIBUTORS.md:10]:
```suggestion
* [Dipin Nair](https://github.com/dipinknair)
```

**Review by RobPasMue — APPROVED** [review]

File: CONTRIBUTORS.md (hunk 1 above)
Comment (Revathyvenugopal162) [reply CONTRIBUTORS.md:14]: Thanks @RobPasMue , i was double checking the names. This is so helpful.

File: CONTRIBUTORS.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+## Project Lead
+* [Revathy Venugopal](https://github.com/Revathyvenugopal162)
+
+## Individual Contributors
+
+* [Alexander Kaszynski](https://github.com/akaszynski)
+* [Andy Grigg](https://github.com/Andy-Grigg)
+* [Dipin Nair](https://github.com/dipinknair)
+* [Dominik Gresch](https://github.com/greschd)
+* [German Martínez](https://github.com/germa89)
```
Comment (germa89) [inline CONTRIBUTORS.md:12]: ❤️

**Review by germa89 — APPROVED** [review]
LGTM. It seems more format-oriented changes.

**Pull request merged**